index.html

```diff
@@ -2,22 +2,24 @@
 <link rel="stylesheet" href="style/style.css">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface specification**
+**OpenPBR Surface**
 
 <center>Zap Andersson, Paul Edmondson, Julien Guertault, Adrien Herubel, Alan King, </center>
 <center>Peter Kutz, Andréa Machizaud, Jamie Portsmouth, Frédéric Servant.</center>
 
-<center>*Version 0.4, 2024-04-30. A joint work of Adobe and Autodesk*</center>
+<center>*Version 0.4, 2024-04-30. A joint work of Adobe and Autodesk &nbsp; <a href="https://github.com/AcademySoftwareFoundation/OpenPBR" title="Redirect to GitHub page"><img src="images/github-mark.png" alt="homepage" width="24" style="vertical-align: middle; position: relative; bottom: 3px"/></a> *</center>
```

Comment:
I see the value of having a link in the document, but I'm not a fan of a using a logo for that link.
I think a full text link would be clearer and look cleaner as well.

Reply:
Really? I find the GitHub logo fairly universal, and a neat way to indicate what the link is.

If we make it a text link, what is the text, and where.

Reply:
Matter of taste I guess; that's a cosmetic detail anyway, let's move on to the rest.